pkg/apis/planetscale/v2/vitessshard_types.go

```diff
@@ -181,6 +188,13 @@ type VitessShardTabletPool struct {
 	// created for this component.
 	Annotations map[string]string `json:"annotations,omitempty"`
 
+	// ExtraFlags can optionally be used to override default flags set by the
+	// operator, or pass additional flags to child components. All entries must be
+	// key-value string pairs of the form "flag": "value". The flag name should
+	// not have any prefix (just "flag", not "-flag"). To set a boolean flag,
+	// set the string value to either "true" or "false".
+	ExtraFlags map[string]string `json:"extraVitessFlags,omitempty"`
```

Comment:
I don't think we need to expose this. We can just invisibly merge the VitessShard-level ExtraVitessFlags into VttabletSpec.ExtraFlags.

Reply:
Adjusted in https://github.com/planetscale/vitess-operator/pull/14/commits/6761ea423399f13542e587676abf9bf4ee58db196.